source/_components/smartthings.light.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: "SmartThings Light"
+description: "Instructions on setting up Samsung SmartThings lights within Home Assistant."
+date: 2018-01-30 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: samsung_smartthings.png
+ha_category: Light
+ha_release: ""
```

Comment:
`0.87`

Reply:
Parent has been tagged and merged - updated.